File: resources/templates/javascript/stub/payload_processor.js
var PayloadProcessingResult = Packages.burp.api.montoya.intruder.PayloadProcessingResult;
var PayloadData = Packages.burp.api.montoya.intruder.PayloadData;
var ByteArray = Packages.burp.api.montoya.core.ByteArray;

function processPayload(montoyaApi, payloadData) {
    var origPayload = java.lang.String(payloadData.currentPayload())
    var processedPayload = "";
    for ( var i =origPayload.length-1 ; i >= 0 ; i-- ) {
            processedPayload += origPayload[i];
    }
    return PayloadProcessingResult.usePayload(ByteArray.byteArray(processedPayload));
}

File: resources/templates/python/stub/payload_processor.py
"""
    Payload processor ( Python )
    - Used by intruder when a payload is to be processed
    - The original payload can be obtained with payloadData.insertionPoint().baseValue()

    Example:
    - This script reverses the string

    Returns:
    A PayloadProcessingResult object with an updated payload
"""
from burp.api.montoya.intruder import PayloadProcessingResult
from burp.api.montoya.intruder import PayloadData
from burp.api.montoya.core import ByteArray

def processPayload(montoyaApi, payloadData):
	return PayloadProcessingResult.usePayload(ByteArray.byteArray(payloadData.insertionPoint().baseValue().toString()[::-1]))